vita3k/host/src/pkg.cpp

```diff
@@ -252,7 +254,10 @@ bool install_pkg(const std::string &pkg, HostState &host, std::string &p_zRIF) {
             LOG_ERROR("The pkg file size is too small, possibly corrupted");
             return false;
         }
-
+        float file_count = byte_swap(pkg_header.file_count);
+        if (i % static_cast<int>(round(file_count / 100.f)) == 0) {
+            *progress = *progress + 0.6;
```

Comment:
Can write more short
```suggestion
            *progress += 0.6;
```

Reply:
Maybe we can just split this to multiple progresses (extract_progress, copy_progress) And do weighted average one it. (progress =  0.2 * extract_progress + 0.8 * copy_progress)